airflow/config_templates/config.yml

```diff
@@ -418,6 +418,18 @@
       type: string
       default: ~
       example: '{"some_param": "some_value"}'
+    - name: database_access_isolation
+      description: Whether components should use Airflow Internal API for DB connectivity.
+      version_added: 2.6.0
+      type: boolean
+      example: ~
+      default: "False"
+    - name: database_api_url
```

Comment:
Should we rather call it "internal_api_url"?
cc: @potiuk

Reply:
Renamed the configuration name

I think we shouldn't reinvent  the wheel there - there are good LB solutions that could be used in front of the multiple Airflow Internal API servers. Unless there is a reason to use L7 LB (not sure about though)